src/cluster/sharding.rst

```diff
@@ -12,290 +12,490 @@
+      "_id": "{name}",
+      "_rev": "1-e13fb7e79af3b3107ed62925058bfa3a",
+      "shard_suffix": [46, 49, 53, 51, 48, 50, 51, 50, 53, 50, 54],
+      "changelog": [
+        ["add", "00000000-1fffffff", "node1@xxx.xxx.xxx.xxx"],
+        ["add", "00000000-1fffffff", "node2@xxx.xxx.xxx.xxx"],
+        ["add", "00000000-1fffffff", "node3@xxx.xxx.xxx.xxx"],
+        …
+        ["add", "00000000-1fffffff", "node4@xxx.xxx.xxx.xxx"]
+      ],
+      "by_node": {
+        "node1@xxx.xxx.xxx.xxx": [
+          "00000000-1fffffff",
+          …
         ],
-        [
-            "add",
-            "80000000-ffffffff",
-            "node2@yyy.yyy.yyy.yyy"
+        …
+        "node4@xxx.xxx.xxx.xxx": [
+          "00000000-1fffffff"
         ]
+      },
+      "by_range": {
+        "00000000-1fffffff": [
+          "node1@xxx.xxx.xxx.xxx",
+          "node2@xxx.xxx.xxx.xxx",
+          "node3@xxx.xxx.xxx.xxx",
+          "node4@xxx.xxx.xxx.xxx"
         ],
-        "by_node": {
-            "node1@xxx.xxx.xxx.xxx": [
-                "00000000-7fffffff",
-                "80000000-ffffffff"
-            ],
-            "node2@yyy.yyy.yyy.yyy": [
-                "00000000-7fffffff",
-                "80000000-ffffffff"
-            ]
-        },
-        "by_range": {
-            "00000000-7fffffff": [
-                "node1@xxx.xxx.xxx.xxx",
-                "node2@yyy.yyy.yyy.yyy"
-            ],
-            "80000000-ffffffff": [
-                "node1@xxx.xxx.xxx.xxx",
-                "node2@yyy.yyy.yyy.yyy"
-            ]
-        }
+        …
+      }
     }
 
-After PUTting this document, it's like magic: the shards are now on node2 too!
-We now have ``n=2``!
+Now you can ``PUT`` this new metadata:
 
-If the shards are large, then you can copy them over manually and only have
-CouchDB sync the changes from the last minutes instead.
+.. code:: bash
 
-.. _cluster/sharding/move:
+    $ curl -X PUT $COUCH_URL:5986/_dbs/{name} -d '{...}'
+
+Monitor internal replication to ensure up-to-date shard(s)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After you complete the previous step, as soon as CouchDB receives a
+write request for a shard on the target node, CouchDB will check if the
+target node's shard(s) are up to date. If it finds they are not up to
+date, it will trigger an internal replication job to complete this task.
+You can observe this happening by triggering a write to the database
+(update a document, or create a new one), while monitoring the
+``/_node/<nodename>/_system`` endpoint, which includes the
+``internal_replication_jobs`` metric.
+
+Once this metric has returned to the baseline from before you wrote the
+document, or is zero (0., it is safe to proceed.
+
+Clear the target node's maintenance mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can now let the node back into the rotation for load balancing by
+putting ``"false"`` to the maintenance mode configuration endpoint, just
+as in step 2.
```

Comment:
I think this is a little misleading -- the reason the node is in mm is because it shouldn't be servicing requests for the data it's hosting, because it had an out of date shard replica of the shard we're moving. That it was out of the LB's rotation is a side-effect rather than intended effect in this particular task.

So I think this could be something like "You can now let the node start servicing data requests by putting...".